bool skewer_action_flag

---

bool success